config/jobs/kubernetes-sigs/service-apis/service-apis-config.yaml

```diff
@@ -0,0 +1,30 @@
+presubmits:
+  kubernetes-sigs/service-apis:
+  - name: pull-service-apis-verify
+    annotations:
+      testgrid-dashboards: sig-network-service-apis
+      testgrid-tab-name: verify
+    decorate: true
+    path_alias: sigs.k8s.io/service-apis
+    always_run: true
+    skip_report: false
+    spec:
+      containers:
+      - image: golang:1.13
+        command:
+        - make
+        args:
+        - verify
```

Comment:
@bowei `make verify` isn't implemented, I've tried to implement it today but I gave up because there are is not straightforward to understand how it works
https://github.com/kubernetes-sigs/service-apis/issues/2

Reply:
It suffices to check this in for just `make`. 
We can work on various `verify` bits one piece at a time.

Reply:
ok, I will remove it and we go with make in this first iteration

Reply:
done